src/hotspot/share/memory/arena.cpp

```diff
@@ -31,12 +31,30 @@
 #include "nmt/memTracker.inline.hpp"
 #include "runtime/os.hpp"
 #include "runtime/task.hpp"
-#include "runtime/threadCritical.hpp"
 #include "runtime/trimNativeHeap.hpp"
 #include "utilities/align.hpp"
 #include "utilities/debug.hpp"
 #include "utilities/ostream.hpp"
 
+// One global static mutex for chunk pools.
+// It is used very early in the vm initialization, in allocation
+// code and other areas.  For many calls, the current thread has not
+// been created so we cannot use Mutex.
+static PlatformMutex* _global_chunk_pool_mutex = nullptr;
```

Comment:
This isn't a class data member, so a leading underscore shouldn't be used.
https://github.com/openjdk/jdk/blame/b6b5ac1ef9042ed62a8358aa6943b8dc87dcf0ab/doc/hotspot-style.md#L260-L262

Maybe this falls under:
https://github.com/openjdk/jdk/blame/b6b5ac1ef9042ed62a8358aa6943b8dc87dcf0ab/doc/hotspot-style.md#L223-L224
though I think that's rare usage for file-scoped static variables.  Just
dropping the leading underscore would probably be the most consistent with
other code.

Reply:
Hm TIL - I've always used underscores for global static variables.